src/db/logger.cr

```diff
@@ -0,0 +1,30 @@
+require "logger"
+
+module DB
+  # :nodoc:
+  class Logger
+    @logger : Proc(String, Void)?
+    @enabled = false
+    @default_logger = ::Logger.new(STDOUT)
```

Comment:
Why not just use `@logger = ::Logger.new(STDOUT)` instead? It's simpler and I don't see the need for supporting `Logger.logger = nil`.